dataprep/clean/clean_text.py

```diff
@@ -0,0 +1,441 @@
+        {"operator": "remove_digits"},
+        {"operator": "remove_html"},
+        {"operator": "remove_urls"},
+        {"operator": "remove_punctuation"},
+        {"operator": "remove_accents"},
+        {"operator": "remove_stopwords", "parameters": {"stopwords": None}},
+        {"operator": "remove_whitespace"},
+    ]
+
+
+def _get_default_pipeline(
+    stopwords: Optional[Set[str]] = None,
+) -> List[Callable[..., Any]]:
+    """
+    Return a list of functions defining the default pipeline.
+    """
+    return [
+        _fillna,
+        _lowercase,
+        _remove_digits,
+        _remove_html,
+        _remove_urls,
+        _remove_punctuation,
+        _remove_accents,
+        lambda x: _remove_stopwords(x, stopwords),
+        _remove_whitespace,
+    ]
+
+
+def _get_custom_pipeline(pipeline: List[Dict[str, Any]]) -> List[Callable[..., Any]]:
+    """
+    Return a list of functions defining a custom pipeline.
+    """
+    func_dict = _get_func_dict()
+    custom_pipeline: List[Callable[..., Any]] = []
+
+    for component in pipeline:
+        # Check whether function is built in or user defined
+        operator = (
+            func_dict[component["operator"]]
+            if isinstance(component["operator"], str)
+            else component["operator"]
+        )
+        # Append the function to the pipeline
+        # If parameters are specified, create a partial function to lock in
+        # the values and prevent them from being overwritten in subsequent loops
+        if "parameters" in component:
+            custom_pipeline.append(_wrapped_partial(operator, component["parameters"]))
+        else:
+            custom_pipeline.append(operator)
+
+    return custom_pipeline
+
+
+def _get_func_dict() -> Dict[str, Callable[..., Any]]:
+    """
+    Return a mapping of strings to function names.
+    """
+    return {
+        "fillna": _fillna,
+        "lowercase": _lowercase,
+        "sentence_case": _sentence_case,
+        "title_case": _title_case,
+        "uppercase": _uppercase,
+        "remove_accents": _remove_accents,
+        "remove_bracketed": _remove_bracketed,
+        "remove_digits": _remove_digits,
+        "remove_html": _remove_html,
+        "remove_prefixed": _remove_prefixed,
+        "remove_punctuation": _remove_punctuation,
+        "remove_stopwords": _remove_stopwords,
+        "remove_urls": _remove_urls,
+        "remove_whitespace": _remove_whitespace,
+        "replace_bracketed": _replace_bracketed,
+        "replace_digits": _replace_digits,
+        "replace_prefixed": _replace_prefixed,
+        "replace_punctuation": _replace_punctuation,
+        "replace_stopwords": _replace_stopwords,
+        "replace_text": _replace_text,
+        "replace_urls": _replace_urls,
+    }
+
+
+def _fillna(text: str, value: Any = np.nan) -> dd.Series:
+    """
+    Replace all null values with NaN (default) or the supplied value.
+    """
+    return value if text in NULL_VALUES else text
```

Comment:
I just gave it a try. Since `series.replace()` doesn't take a set in `to_replace`, I used `column.replace(list(NULL_VALUES), value)`. It seems to work just as well, maybe even a little better.